### PR TITLE
ICMSLST-641 Add Checklist for Wood Quota applications.

### PR DESCRIPTION
**Pull request by MattHolmes123**

### CODE REVIEW

File: web/domains/case/_import/wood/models.py [hunk 1]
```diff
@@ -34,3 +34,61 @@ class WoodQuotaApplication(ImportApplication):
...
+
+
+class WoodQuotaChecklist(models.Model):
+    class Response(models.TextChoices):
+        yes = ("yes", "Yes")
+        no = ("no", "No")
+        not_applicable = ("n/a", "N/A")
+
+    import_application = models.ForeignKey(
+        WoodQuotaApplication, on_delete=models.PROTECT, related_name="checklists"
```
Comment (oskusalerma) [inline web/domains/case/_import/wood/models.py:46]: related_name is too generic, how about woodquota_checklists?

File: web/domains/case/_import/wood/models.py [hunk 2]
```diff
@@ -34,3 +34,61 @@ class WoodQuotaApplication(ImportApplication):
...
+    )
+
+    sigl_wood_application_logged = models.BooleanField(
+        default=False,
+        verbose_name="Log on to SIGL Wood via this screen. Processing done on SIGL Wood.",
+    )
+
+    case_update = models.CharField(
```
Comment (oskusalerma) [inline web/domains/case/_import/wood/models.py:54]: could these yes/no/na fields be nullable booleans?

**Review by oskusalerma — DISMISSED** [review]

File: web/domains/case/_import/wood/models.py (hunk 2 above)
Comment (MattHolmes123) [reply web/domains/case/_import/wood/models.py:54]: No I thought that in the other checklist I did.
N/a is a valid required field

File: web/domains/case/_import/wood/models.py (hunk 1 above)
Comment (MattHolmes123) [reply web/domains/case/_import/wood/models.py:46]: The equivalent related names on the two other models are "checklists".

Comment (MattHolmes123) [reply web/domains/case/_import/wood/models.py:46]: I don't feel to strongly either way, I can change it if you want.
@kevinetienne what do you think?

**Review by kevinetienne — APPROVED** [review]

**Pull request merged**